### PR TITLE
[SofaLoader] ADD support to load VTK polylines in legacy formated files

### PR DESCRIPTION
**Pull request by untereiner**
- Add support to load VTK polylines in legacy formated files
- used the new astyle formating onto the three files

______________________________________________________

This PR: 
- [x] builds with SUCCESS for all platforms on the CI.
- [x] does not generate new warnings.
- [x] does not generate new unit test failures.
- [x] does not generate new scene test failures.
- [x] does not break API compatibility.
- [x] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by untereiner** [comment]
[ci-build][with-scene-tests]

**Comment by untereiner** [comment]
there are several lines at several places.... a bit shitting to put them here... search for Polyline

**Comment by untereiner** [comment]
MeshLoader.cpp (ligne 690)
MeshVTKLoader.h (lignes 75 - 135 - 194)
MeshVTKLoader.cpp (lignes 265 - 373 - 535)

**Comment by guparan** [comment]
What astyle command did you launch to get this diff?
I don't have the same output when I do
`astyle --options=tools/astyle/sofa.astyle SofaKernel/modules/SofaLoader/MeshVTKLoader.h SofaKernel/modules/SofaLoader/MeshVTKLoader.cpp  SofaKernel/modules/SofaLoader/MeshVTKLoader.cpp`

**Comment by guparan** [comment]
My bad, I shouldn't trust GitHub blindly.
Here is the real diff of this PR: https://github.com/guparan/sofa/commit/3f9d02407598ca760e79682a15d4cebbdc84d85d

File: SofaKernel/modules/SofaLoader/MeshVTKLoader.cpp [hunk 1]
```diff
@@ -527,6 +532,7 @@ bool MeshVTKLoader::setInputsMesh()
         delete reader->inputCellTypes;
     }
 
+    d_polygons.endEdit();
```
Comment (guparan) [inline SofaKernel/modules/SofaLoader/MeshVTKLoader.cpp]: Did you mean `d_polylines`?

Comment (untereiner) [reply SofaKernel/modules/SofaLoader/MeshVTKLoader.cpp]: yep, I'll change it

**Pull request merged**